source/opt/struct_cfg_analysis.h

```diff
@@ -74,6 +74,9 @@ class StructuredCFGAnalysis {
   // construct.
   uint32_t LoopContinueBlock(uint32_t bb_id);
 
+  // Returns the loop nesting depth of |bb_id|.
```

Comment:
I think it would be good to elaborate slightly more on what this means, and be clear that it is the depth with respect to the current function.

"Returns the loop nesting depth of |bb_id| within its function.  That is, the number of loop constructs in which |bb_id| is contained.  Note that, as per other functions in StructuredCFGAnalysis, a loop header is *not* regarded as being part of the loop that it heads, so that e.g. the nesting depth of an outer-most loop header is 0."

Reply:
Done (also in #3754)